adblock_social_filters/adblock_social_list.txt

```diff
@@ -1101,7 +1101,7 @@ bitcoin-online.pl##.social-color
 bithub.pl##.st-wykop, .st-steemit
 bitserwis.com.pl###node-490
 biuroprasowe.pl##.twt, .lin
-biurorachunkowe-swidnica.com.pl,jampol.eu,materialybudowlanekrokowa.pl,mckenzie.pl,oknajachimczak.com,sklep.akces-plexi.pl,sklep.hart-metale.pl,uzdrowiska-klodzkie.pl,uzdrowiska-pgu.pl,www.hanbud.com.pl##.kac-sm-panel
+biurorachunkowe-swidnica.com.pl,jampol.eu,materialybudowlanekrokowa.pl,mckenzie.pl,oknajachimczak.com,sklep.akces-plexi.pl,sklep.audiofil.eu,sklep.hart-metale.pl,uzdrowiska-klodzkie.pl,uzdrowiska-pgu.pl,www.hanbud.com.pl##.kac-sm-panel
```

Comment:
To tylko dziwne kolorowanie gdzie jest zmiana, nikt nie modyfikował `hart`, powinno do Github naprawić.